scale/queue/job_exe.py

```diff
@@ -45,6 +45,7 @@ def create_job_exe_model(self, framework_id, when):
         job_exe.batch_id = self._queue.batch_id
         job_exe.exe_num = self._queue.exe_num
         job_exe.timeout = self._queue.timeout
+        job_exe.docker_image = self._queue.docker_image
```

Comment:
This either needs to have a fall back or we need to populate the `docker_image` field in the `queue` table with a migration prior to execution. Otherwise existing jobs on the queue will fail with blank Docker image names,